.eslintrc.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   plugins: ['@typescript-eslint'],
   rules: {
-    'comma-dangle': 'off'
+    'comma-dangle': 'off',
+    'react-native-a11y/has-accessibility-hint': 'off'
```

Comment:
Why this change?

Reply:
But this is a recommended best practice - hence the question.

I dont believe we have disabled this on other apps

Reply:
https://github.com/FormidableLabs/eslint-plugin-react-native-a11y/blob/master/docs/rules/has-accessibility-hint.md#has-accessibility-hint

Reply:
I'd encourage maintaining use of this rule